packages/alfa-rules/src/sia-r83/rule.ts

```diff
@@ -214,6 +243,13 @@ function horizontallyClippingAncestor(
           return None;
         }
 
+        if (test(usesFontRelativeMediaRule(device, isWidth), element)) {
+          // The element uses a (font relative) media rule, and we can't guess what
+          // the page would like upon resizing and triggering a different media
+          // query, so we just accept it as good enough
+          return None;
+        }
```

Comment:
It looks like we are checking for a 2nd time - the check on lines 195-201 appears the same

Reply:
Yes, that's due to a specificity of horizontal wrapping which does not only depend on `overflow-x` but also on `text-overflow` and `white-space`. The later two sort of only work on the parent of the text (but not fully, #955 ).
So, we have a first `horizontallyClipper` on the parent of text, if text overflows its parent, we look for a `horizontallyClippingAncestor` considering that it is now content and not text (so `white-space` doesn't act anymore).
Both of these are quite similar (might be room for improvement) and both need to accept a font relative media query. The former calls the later which then recurse up the DOM tree.